test/server_test.py

```diff
@@ -64,6 +67,20 @@ def tmp_dirs(tmpdir):
     tmpdir.mkdir("conf").mkdir("conf.d")
 
 
+@pytest.fixture
+def tmp_ticket(tmpdir):
+    img_size = 16 * KiB
+    img_path = testutil.create_tempfile(tmpdir, "disk.raw", size=img_size)
+    ticket_path = tmpdir.join("file.json")
+    ticket_path.write(
+        json.dumps(
+            testutil.create_ticket(
+                uuid="file",
+                size=img_size,
+                url=f'file://{img_path}')))
```

Comment:
Trying to avoid temporary variables makes this code harder to read and write.

Why not:

    ticket = testutil.create_ticket(
        uuid="file", size=img_size, url=f'file://{img_path}')
    ticket.path.write(json.dumps(ticket)))